tests/smoke-tests/pom.xml

```diff
@@ -1199,7 +1199,7 @@
                         <arg>--name</arg>
                         <arg>cluster1</arg>
                         <arg>--clustered</arg>
-                        <arg>--staticCluster</arg>
+                        <arg>--static-cluster</arg>
```

Comment:
Similarly seems like its unrelated - not even for the test in question. Would also likely be clearer if separated out, why this one is being changed and the other server for the test isnt, that they are synonym options.